connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -57,24 +61,31 @@ async def validate(
         )
         # vals is a validations dict for each required trust root containing validated
         # digests and errors
-        vals = self.__get_pinned_trust_roots(trustRoot, required, threshold)
+        vals = self.__get_pinned_trust_roots(
+            trustRoot, required, threshold, verifyInTransparencyLog
+        )
 
-        # prepare executor
-        num_workers = len(vals)
-        executor = ThreadPoolExecutor(num_workers)
         # prepare tasks
         # a copy of vals dictionaries is passed to concurrent validation to ensure
         # thread-safe execution
         arguments = [(k, v.copy(), str(image)) for k, v in vals.items()]
-        futures = [executor.submit(self.__validation_task, *arg) for arg in arguments]
-        for future in futures:
-            vals.update(future.result())
+        results = await asyncio.gather(
+            *[self.__validation_task(*args) for args in arguments]
+        )
+        for result in results:
+            vals.update(result)
 
         return CosignValidator.__apply_policy(
             vals=vals, threshold=threshold, required=required
         )
 
-    def __get_pinned_trust_roots(self, tr_name: str, required: list, threshold: int):
+    def __get_pinned_trust_roots(
+        self,
+        tr_name: str,
+        required: list,
+        threshold: int,
+        verifyTransparencyLog: bool,  # pylint: disable=invalid-name
```

Comment:
Why the naming diff between `verifyInTransparencyLog` and `verifyTransparencyLog`?

Reply:
Also, couldn't this be the pythonic_snake_name?